src/ripple/ledger/tests/PaymentSandbox_test.cpp

```diff
@@ -290,10 +291,50 @@ class PaymentSandbox_test : public beast::unit_test::suite
                 expect (pv.balanceHook (alice, gw, hugeAmt) != tinyAmt);
         }
     }
+    void testReserve()
+    {
+        testcase ("Reserve");
+        using namespace jtx;
+
+        beast::Journal dj;
+
+        auto accountFundsXRP = [&dj](
+            ReadView const& view, AccountID const& id) -> XRPAmount {
+            FreezeHandling const freezeHandling = fhZERO_IF_FROZEN;
+            return toAmount<XRPAmount> (accountHolds (
+                view, id, xrpCurrency (), xrpAccount (), freezeHandling, dj));
```

Comment:
I think you can get rid of the `freeezeHandling` local and just use `fhZERO_IF_FROZEN` in place in the call to `accountHolds`.  Just a thought, not a big deal.